DQM/EcalMonitorTasks/src/OccupancyTask.cc

```diff
@@ -25,6 +24,11 @@ namespace ecaldqm {
     return false;
   }
 
+  void OccupancyTask::beginRun(edm::Run const&, edm::EventSetup const& _es) {
+    _es.get<EcalLaserDbRecord>().get(laser);
```

Comment:
This access of data from ES without token is deprecated. Please follow the instructions in this [twiki](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#In_ED_module) to update accessing the data from  ES.

Reply:
Thanks. It is updated now.